src/autofill.js

```diff
@@ -3,6 +3,18 @@
         if (!window.isSecureContext) return
 
         const listenForGlobalFormSubmission = require('./Form/listenForFormSubmission')
+        const {isAndroid} = require('./autofill-utils')
+        const {processConfig} = require('@duckduckgo/content-scope-scripts/src/apple-utils')
+
+        if (!isAndroid) {
+            // @ts-ignore - variable populated during injection
+            const privacyConfig = processConfig($CONTENT_SCOPE$, $USER_UNPROTECTED_DOMAINS$, $USER_PREFERENCES$)// eslint-disable-line no-undef
+            const site = privacyConfig.site
+            if (site.isBroken || site.isAllowlisted || !site.enabledFeatures.includes('autofill')) {
```

Comment:
I'm not sure if we should stop execution if `site.isBroken` or `site.isAllowlisted`. These signal problems with one of the protections. It shouldn't affect autofill. That is how we did it for extensions: `site?.brokenFeatures?.includes('autofill')`. You can see it a few lines below in this same file.

Reply:
In this case `site.isBroken` means the site is on the global unprotected list. In this case we normally disable all features on a page. We also normally disable all features if a site is allow listed. This is how remote config features are normally handled. If we want to deviate from that in this case we should probably open a discussion with stake holders.